src/tests/class_tests/smartpeak/source/Utilities_test.cpp

```diff
@@ -388,10 +409,10 @@ BOOST_AUTO_TEST_CASE(getPathnameContent)
   const std::array<std::vector<std::string>, 4> c = Utilities::getPathnameContent(pathname);
 
   // number of items in the pathname, taking .gitignore into account
-  BOOST_CHECK_EQUAL(c[0].size(), 45);
```

Comment:
This is a little bit boring to have to change that test every time we add files in the test directory.

Reply:
we can remove it in this PR, much better would be a test that includes all related IO file operations including assertions if directory contents, etc.

Reply:
I will create a task, it's better to focus on your task as defined for now, it's already a lot.